charts/barman/Chart.yaml

```
@@ -2,16 +2,16 @@ apiVersion: v2
 name: barman
 type: application
 description: Chart for Barman PostgreSQL Backup and Recovery Manager
-version: 0.10.0
-appVersion: "v2.19"
+version: 0.10.1
```

This feels like more than a bugfix so a minor bump would be very much ok.

```suggestion
version: 0.11.0
```